ocean_lib/models/v4/erc721_token.py

```diff
@@ -108,16 +121,6 @@ def add_to_metadata_list(self, allowed_address: str, from_wallet: Wallet) -> str
     def add_manager(self, manager_address: str, from_wallet: Wallet) -> str:
         return self.send_transaction("addManager", (manager_address,), from_wallet)
 
-    def add_to_725_store_list(self, allowed_address: str, from_wallet: Wallet) -> str:
```

Comment:
Why did you remove these functions?

Reply:
Check https://github.com/oceanprotocol/ocean.py/blob/v4main/ocean_lib/models/v4/erc721_token.py#L98 and https://github.com/oceanprotocol/ocean.py/blob/v4main/ocean_lib/models/v4/erc721_token.py#L111 are duplicates